frontend/public/components/routes/create-route.tsx

```diff
@@ -276,11 +364,37 @@ export class CreateRoute extends React.Component<null, CreateRouteState> {
           <div className="form-group co-create-route__service">
             <label className="co-required" htmlFor="service">Service</label>
             {(loaded && _.isEmpty(serviceOptions)) && <Alert isInline className="co-alert co-create-route__alert" variant="info" title="No services">There are no services in your project to expose with a route.</Alert>}
-            {(loaded && !_.isEmpty(serviceOptions)) && <Dropdown items={serviceOptions} title={service ? serviceOptions[service.metadata.name] : 'Select a service'} dropDownClassName="dropdown--full-width" id="service" onChange={this.changeService} describedBy="service-help" /> }
+            {(loaded && !_.isEmpty(serviceOptions)) && <Dropdown items={availableServiceOptions} title={service ? serviceOptions[service.metadata.name] : 'Select a service'} dropDownClassName="dropdown--full-width" id="service" onChange={this.changeService} describedBy="service-help" /> }
             <div className="help-block" id="service-help">
               Service to route to.
             </div>
           </div>
+          {alternateServicesList.length > 0 &&
+            <React.Fragment>
+              <div className="form-group co-create-route__weight">
+                <label htmlFor="weight">Weight</label>
+                <div>
+                  <input className="pf-c-form-control"
+                    type="number"
+                    onChange={this.handleWeightChange}
+                    value={this.state.weight}
+                    id="weight"
+                    name="weight"
+                    aria-describedby="weight-help" />
+                </div>
+                <div className="help-block" id="weight-help">
+                  A number between 0 and 255 that depicts relative weight compared with other targets.
+                </div>
+              </div>
+              {alternateServicesList}
+            </React.Fragment>}
+          {(alternateServicesList.length < 3 && service) && <Button
```

Comment:
Why do we check the length? You should be able to add any number of alternate backends

Reply:
@spadgett thanks for the review! Yes the length check is due to description in the api docs for alternate back ends as follows: 
"alternateBackends allows up to 3 additional backends to be assigned to the route. Only the Service kind is allowed, and it will be defaulted to Service. Use the weight field in RouteTargetReference object to specify relative preference."

Reply:
OK, I didn't realize that! Thanks, disregard my comment